A document about our Zulip channel (#19)
anitab-org/documentation: zulip.md

# Zulip

##  Getting started on Zulip:

### If you’re new here, here’s what you can do to get started with this community:

> 1.  :heavy_check_mark:  [Introduce yourself on #newcomers in the #newcomers > Introductions topic and tell us your motivations for joining so we can get to know you and help you on your journey](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers)

Reviewer: ```suggestion
1.  :heavy_check_mark:  Introduce yourself on [#newcomers](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) stream in the [#newcomers > Introductions](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers/topic/Introductions) topic and tell us your motivations for joining so we can get to know you and help you on your journey.
```

> 2.  :calendar:  [Follow our Open Sessions in Calendar Events topic on #announcements or through our AnitaB.org Open Source calendar (PST:clock1:). These are voice calls you can participate in and get to know a bit more our community and projects](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles)

Reviewer: ```suggestion
2.  :calendar: Follow our Open Sessions in Calendar Events topic on [#announcements](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements) or through our AnitaB.org Open Source calendar ([PST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles) || [EST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/New_York) || [GMT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=GMT) || [WAT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Africa/Lagos) || [IST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Asia/Colombo) || [AEDT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Australia/Sydney)). These are voice calls you can participate in and get to know a bit more our community and projects.
```

##   Learning Resources :eyeglasses:
### To keep this community safe for you and others.

> 1.  [Code of Conduct](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/code_of_conduct.md)     
> 
> 2.  [Reporting Guidelines](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/reporting_guidelines.md) 
> 
> 3.  [To get Acquainted with Zulip](https://zulipchat.com/help/getting-started-with-zulip)  

Reviewer: ```suggestion
1.  [Code of Conduct](/Contributing/CODE_OF_CONDUCT.md)     
2.  [Reporting Guidelines](/Contributing/REPORTING_GUIDELINES.md) 
3.  [Zulip official documentation on how to getting started](https://zulipchat.com/help/getting-started-with-zulip)  
```

##  Streams:point_down:
    

> 1. Category baseds: These are streams based on a specific type of contribution (e.g.: quality assurance, coding, documentation, etc). Join streams of your interest such as:

Reviewer: can you make these lists non enumerated? So instead of using 1, 2, 3, using "*" or "-" to define each entry. This is because there is no streams more important then other, they are all at the same relevance level

Author: Definitely, working on it right now.

    1.  [Coding](https://anitab-org.zulipchat.com/#narrow/stream/216321-coding):computer:
    2.  [Design](https://anitab-org.zulipchat.com/#narrow/stream/216323-design):art:

>     3.  [Documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentatio):notebook_with_decorative_cover:

Reviewer: ```suggestion
    3.  [Documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentation) :notebook_with_decorative_cover:
```

Reviewer: can you add a space character between the emojis and the text for all the places where you have emojis and text. It can make it a bit more readable

Author: Definitely, working on it right now.

    4.  [Outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research):microscope:
    5.  [Quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance):heavy_check_mark:

2. Project Specific: You can join also project-specific streams, such as: 

>     1.  [Mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system):mortar_board:

Reviewer: ```suggestion
    1.  [Mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
```

        1.  [Mentorship-backend](https://github.com/anitab-org/mentorship-backend)
        2.  [Mentorship-flutter](https://github.com/anitab-org/mentorship-flutter)
        3.  [Mentorship-android](https://github.com/anitab-org/mentorship-android)
        4.  [Mentorship-ios](https://github.com/anitab-org/mentorship-ios)

>     2.  [Open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs):confetti_ball:

Reviewer: ```suggestion
    2.  [Open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
```

>     1.  [Coding](https://anitab-org.zulipchat.com/#narrow/stream/216321-coding) :computer:
>     2.  [Design](https://anitab-org.zulipchat.com/#narrow/stream/216323-design) :art:
>     3.  [Documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentation) :notebook_with_decorative_cover:    
>     4.  [Outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research) :microscope:
>     5.  [Quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance) :heavy_check_mark:

Reviewer: ```suggestion
    1.  [#coding](https://anitab-org.zulipchat.com/#narrow/stream/216321-coding) :computer:
    2.  [#design](https://anitab-org.zulipchat.com/#narrow/stream/216323-design) :art:
    3.  [#documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentation) :notebook_with_decorative_cover:    
    4.  [#outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research) :microscope:
    5.  [#quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance) :heavy_check_mark:
```

>     * [Mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system) :mortar_board:

Reviewer: ```suggestion
    * [Mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
```

>         *   [Mentorship-backend](https://github.com/anitab-org/mentorship-backend)

Reviewer: ```suggestion
        *   [anitab-org/mentorship-backend](https://github.com/anitab-org/mentorship-backend)
```

        1.  [Open-source-programs-web](https://github.com/anitab-org/open-source-programs-web)
        2.  [Open-source-hack-website](https://github.com/anitab-org/Open-source-hack-website)
        3.  [Open-source-programs-backend](https://github.com/anitab-org/open-source-programs-backend)

>     3.  [Portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal):tada:

Reviewer: ```suggestion
    3.  [Portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal)
```

>         *   [Mentorship-flutter](https://github.com/anitab-org/mentorship-flutter)

Reviewer: ```suggestion
        *   [anitab-org/mentorship-flutter](https://github.com/anitab-org/mentorship-flutter)
```

        1.  [Anitab-org/portal](https://github.com/anitab-org/portal)

>     4.  [STEM-diverse-tv](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv):tv:

Reviewer: ```suggestion
    4.  [STEM-diverse-tv](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv)
```

>         *   [Mentorship-backend](https://github.com/anitab-org/mentorship-backend)
>         *   [Mentorship-flutter](https://github.com/anitab-org/mentorship-flutter)
>         *   [Mentorship-android](https://github.com/anitab-org/mentorship-android)
>         *   [Mentorship-ios](https://github.com/anitab-org/mentorship-ios)

Reviewer: ```suggestion
        *   [anitab-org/mentorship-backend](https://github.com/anitab-org/mentorship-backend)
        *   [anitab-org/mentorship-flutter](https://github.com/anitab-org/mentorship-flutter)
        *   [anitab-org/mentorship-android](https://github.com/anitab-org/mentorship-android)
        *   [anitab-org/mentorship-ios](https://github.com/anitab-org/mentorship-ios)
```

>     *  [Open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) :confetti_ball:

Reviewer: ```suggestion
    *  [Open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
```

        1.  [Anitab-org/stem-diverse-tv](https://github.com/anitab-org/stem-diverse-tv)

>     5.  [VMS](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms:):sparkler:

Reviewer: ```suggestion
    5.  [VMS](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms:)
```

        1. [Anitab-org/vms](https://github.com/anitab-org/vms)

>     6.  [Anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio):octocat:

Reviewer: Bridge in Tech project is missing here, can you please add it as well.

>     6.  [Anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio):octocat:

Reviewer: ```suggestion
    6.  [Anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)
```

>         *  [Open-source-programs-web](https://github.com/anitab-org/open-source-programs-web)
>         *  [Open-source-hack-website](https://github.com/anitab-org/Open-source-hack-website)
>         *  [Open-source-programs-backend](https://github.com/anitab-org/open-source-programs-backend)

Reviewer: ```suggestion
        *  [anitab-org/open-source-programs-web](https://github.com/anitab-org/open-source-programs-web)
        *  [anitab-org/open-source-programs-backend](https://github.com/anitab-org/open-source-programs-backend)
```
the open source hack website repo is not related to this channel

>     *  [Portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal) :tada:

Reviewer: ```suggestion
    *  [Portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal)
```

>         *   [Anitab-org/portal](https://github.com/anitab-org/portal)

Reviewer: ```suggestion
        *   [anitab-org/portal](https://github.com/anitab-org/portal)
```

>         *   [Anitab-org/stem-diverse-tv](https://github.com/anitab-org/stem-diverse-tv)

Reviewer: ```suggestion
        *   [anitab-org/stem-diverse-tv](https://github.com/anitab-org/stem-diverse-tv)
```

>     *  [VMS](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms:)        

Reviewer: ```suggestion
    *  [#vms](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)        
```

        1.  [Anitab-org/anitab-org.github.io](https://github.com/anitab-org/anitab-org.github.io)

3.  General: You can also check our more general streams:
    1.  [Announcements](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements):microphone:
    2.  [Celebrate](https://anitab-org.zulipchat.com/#narrow/stream/223068-celebrate):sparkler:
    3.  [General](https://anitab-org.zulipchat.com/#narrow/stream/212722-general):newspaper:

>     4.  [Oppoturnites](https://anitab-org.zulipchat.com/#narrow/stream/223069-opportunities):stars:

Reviewer: ```suggestion
    4.  [Opportunities](https://anitab-org.zulipchat.com/#narrow/stream/223069-opportunities) :stars:
```

>         *   [Anitab-org/vms](https://github.com/anitab-org/vms)

Reviewer: ```suggestion
        *   [anitab-org/vms](https://github.com/anitab-org/vms)
```

>     *   [Anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)        

Reviewer: ```suggestion
    *   [#anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)        
```

>         *   [Anitab-org/anitab-org.github.io](https://github.com/anitab-org/anitab-org.github.io)

Reviewer: ```suggestion
        *   [anitab-org/anitab-org.github.io](https://github.com/anitab-org/anitab-org.github.io)
```

>     1.  [Announcements](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements) :microphone:
>     2.  [Celebrate](https://anitab-org.zulipchat.com/#narrow/stream/223068-celebrate) :sparkler:
>     3.  [General](https://anitab-org.zulipchat.com/#narrow/stream/212722-general) :newspaper:
>     4.  [Opportunities](https://anitab-org.zulipchat.com/#narrow/stream/223069-opportunities) :stars:
>     5.  [Questions](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions) :question:
>     6.  [Newcomers](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) :new:  

Reviewer: ```suggestion
    1.  [#announcements](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements) :microphone:
    2.  [#celebrate](https://anitab-org.zulipchat.com/#narrow/stream/223068-celebrate) :sparkler:
    3.  [#general](https://anitab-org.zulipchat.com/#narrow/stream/212722-general) :newspaper:
    4.  [#opportunities](https://anitab-org.zulipchat.com/#narrow/stream/223069-opportunities) :stars:
    5.  [#questions](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions) :question:
    6.  [#newcomers](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) :new:  
```

        *   [anitab-org/mentorship-ios](https://github.com/anitab-org/mentorship-ios)

    *   [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) 
        *   [anitab-org/open-source-programs-web](https://github.com/anitab-org/open-source-programs-web)
        *   [anitab-org/open-source-programs-backend](https://github.com/anitab-org/open-source-programs-backend)
        
    *   [#portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal) 
        *   [anitab-org/portal](https://github.com/anitab-org/portal)

    *   [#stem-diverse-tv](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv)        
        *   [anitab-org/stem-diverse-tv](https://github.com/anitab-org/stem-diverse-tv)
        
    *  [#vms](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)                
        *   [anitab-org/vms](https://github.com/anitab-org/vms)

    *   [#anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)                
        *   [anitab-org/anitab-org.github.io](https://github.com/anitab-org/anitab-org.github.io)

>         

Reviewer 2: Good job @kakabisht , very easy to read!
You can add the following, which is one of the new AnitaB.org projects from GSoC:
* [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech)
   * [anitab-org/bridge-in-tech-backend](https://github.com/anitab-org/bridge-in-tech-backend)
   * [anitab-org/bridge-in-tech-web](https://github.com/anitab-org/bridge-in-tech-web)

Author: HI, @foongminwong fixed the the request, let me know if i could further modify something.

2.   :calendar: Follow our Open Sessions in Calendar Events topic on [#announcements](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements) or through our AnitaB.org Open Source calendar ([PST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles) || [EST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/New_York) || [GMT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=GMT) || [WAT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Africa/Lagos) || [IST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Asia/Colombo) || [AEDT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Australia/Sydney)). These are voice calls you can participate in and get to know a bit more our community and projects.

##   Learning Resources :eyeglasses:

> ### To keep this community safe for you and others.

Reviewer 3: Is `safe` correct term to use?

Author: Hi @jhalak27 , I have fixed the typos,  Let me know if I should use the term safe or not.

Reviewer 3: Maybe 'helpful'. Not sure.
@isabelcosta @foongminwong what you suggest?

Reviewer 2: Suggestion: `These are voice call sessions you can participate to learn more about our community and projects.`

Reviewer 2: > Maybe 'helpful'. Not sure.
> @isabelcosta @foongminwong what you suggest?

Hmm I'm also not sure what word can be used, or we could remove line 13?

1.  [Code of Conduct](/Contributing/CODE_OF_CONDUCT.md)

2.  [Reporting Guidelines](/Contributing/REPORTING_GUIDELINES.md) 

3.  [Zulip official documentation on how to getting started](https://zulipchat.com/help/getting-started-with-zulip)  

##  Streams :point_down:
    

> 1. Category baseds: These are streams based on a specific type of contribution (e.g.: quality assurance, coding, documentation, etc). Join streams of your interest such as:

Reviewer 3: typo
```suggestion
1. Category based: These streams are based on a specific type of contribution (e.g.: quality assurance, coding, documentation, etc). Join streams of your interest such as:
```

    1.  [#coding](https://anitab-org.zulipchat.com/#narrow/stream/216321-coding) :computer:
    2.  [#design](https://anitab-org.zulipchat.com/#narrow/stream/216323-design) :art:
    3.  [#documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentation) :notebook_with_decorative_cover:    
    4.  [#outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research) :microscope:
    5.  [#quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance) :heavy_check_mark:

> 2. Project Specific: You can join also project-specific streams, such as: 

Reviewer 3: ```suggestion
2. Project Specific: You can also join project-specific streams, such as: 
```

# Zulip

##  Getting started on Zulip:

### If you’re new here, here’s what you can do to get started with this community:

> 1.  :heavy_check_mark: Introduce yourself on [#newcomers](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) stream in the [#newcomers > Introductions](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers/topic/Introductions) topic and tell us your motivations for joining so we can get to know you and help you on your journey.

Reviewer 2: Suggestion: `share with us your motivations for joining so we can get to know and help you on the open source journey`
